tests/integration/targets/child_accounts/tasks/main.yaml

```diff
@@ -0,0 +1,31 @@
+- name: child_accounts
+  tags:
+    - never
+    - parent_child
```

Comment:
This makes sure we don't run this test alongside the normal suite because it requires a special testing environment.